api/serializers.py

```diff
@@ -363,5 +363,18 @@ class PlaylistDetailSerializer(PlaylistSerializer):
 
     channel = ChannelSerializer(read_only=True)
 
+    items = serializers.SerializerMethodField(
```

Comment:
If we have ``mediaIds``, this should probably be called ``media`` to match. Or ``mediaIds`` should be called ``itemIds``.